src/openrct2/park/ParkFile.cpp

```diff
@@ -613,6 +613,12 @@ namespace OpenRCT2
                     cs.Write(static_cast<int8_t>(gameState.SavedViewZoom));
                 }
                 cs.ReadWrite(gameState.SavedViewRotation);
+
+                if (os.GetHeader().TargetVersion >= PARK_FILE_CURRENT_VERSION)
```

Comment:
You need to have an explicit version number here not the constant otherwise when the next version comes out it will be wrong.